retroshare-gui/src/gui/feeds/GxsChannelPostItem.cpp

```diff
@@ -392,9 +392,21 @@ void GxsChannelPostItem::fill()
 
 	if(mPost.mThumbnail.mData != NULL)
 	{
-		QPixmap thumbnail;
+		QPixmap thumbnail;	
+		
 		GxsIdDetails::loadPixmapFromData(mPost.mThumbnail.mData, mPost.mThumbnail.mSize, thumbnail,GxsIdDetails::ORIGINAL);
 		// Wiping data - as its been passed to thumbnail.
+		if( thumbnail.width() < 90 ){
+			ui->logoLabel->setMaximumSize(82,108);
+		}
+		else if( thumbnail.width() < 109 ){
+			ui->logoLabel->setMinimumSize(108,108);
+			ui->logoLabel->setMaximumSize(108,108);
+		}
+		else{
+			ui->logoLabel->setMinimumSize(156,108);
+			ui->logoLabel->setMaximumSize(156,108);
+		}
```

Comment:
This is likely to fail, because you give an absolute size everywhere. Try to multiply everything by font height/14.0

Reply:
....or compare the size to that of another widget.

Reply:
without fixed not possible, how to display thumbnail as poster?
People using on channels poster format or 16:9 sometime 1:1
at the moment the thumbnail label uses fixed format on ui without my changes too.
i detect it with the width if its Poster else 16:9 (156,108)

when not set fixed then it will look ugly the label

Reply:
i can remove that changes, and first change all the icons on this pr

Reply:
ok. I'm pretty sure it's possible to do it in a way or another. If you measure how many unites of text height there is, then you can set the size of the image accordingly.

Reply:
the problem people attaching different size/dimension of poster pics format, some is 75x108, 80x108, 82x108... that the reason i set minum and maximum then the empty little background will filled with color.
else you see in one channel lots of different sized qlabels 

Best was at posting you set your right format when attaching thumbnail to the media
and set your default fixed thumbnail format for your channel then we dont need fixed size label.

Reply:
What Im saying is that, instead of typing 108, you should use 108/14.0*font().height(). This way the size adapts to the screen. This is what we do everywhere else.

Reply:
thx i will look on it